supplementary_style_guide/glossary_terms_conventions/general_conventions/r.adoc

```diff
@@ -513,6 +490,19 @@ Always spell out the full product name of the host, and do not capitalize the te
 
 *See also*: xref:red-hat-enterprise-linux-openstack-platform[Red Hat Enterprise Linux OpenStack Platform]
 
+// BxMS: General; kept as is
+[discrete]
+[[red-hat-process-automation-manager]]
+==== image:images/yes.png[yes] Red Hat Process Automation Manager (noun)
+*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). Red Hat Process Automation Manager enables enterprise business and IT users to document, simulate, manage, automate, and monitor business processes and policies. It is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
```

Comment:
```suggestion
*Description*: _Red Hat Process Automation Manager_ is the Red Hat platform for Business Process Management (BPM). By using Red Hat Process Automation Manager, enterprise business and IT users can document, simulate, manage, automate, and monitor business processes and policies. The platform is designed to empower business and IT users to collaborate more effectively, so business applications can be changed more easily and quickly.
```
- Per ISG, avoid anthropomorphic phrasing such as saying that a system "enables" or "allows" humans to to do things. https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism
- `It` in the last sentence has an unclear antecedent.